src/m365/teams/Channel.ts

```diff
@@ -5,4 +5,5 @@ export interface Channel {
   isFavoriteByDefault: boolean | null;
   email: string | null;
   webUrl: string | null;
+  membershipType: string | null;
```

Comment:
We're in the process of moving from custom types to the microsoft-graph-types package where possible. 
So instead of adding to this custom type, let's remove it and import Channel from the microsoft-graph-types package.